module/VuFind/tests/unit-tests/src/VuFindTest/Search/EPF/OptionsTest.php

```diff
@@ -93,7 +93,7 @@ public function testOptions(array $config, $view, $epfView): void
             ]
         );
         $options = new Options($configMgr);
-        $this->assertEquals($view, $options->getView());
-        $this->assertEquals($epfView, $options->getEpfView());
+        // $this->assertEquals($view, $options->getView(), 'View not equal');
```

Comment:
Removing this test unless getView() is actually used.

Reply:
It looks like getView() is a legacy thing that may never have been used, and in any case is poorly named and confusing. I'd be inclined to deprecate or remove it.

Reply:
The Options getView() method seems to be orphaned even in the original commit implementing EDS (c88529d81d4510f45113522825fb9ad8d9322c26), so I'd feel pretty safe just removing it. Maybe it makes sense to do that as a separate PR so it's easily reversible if we somehow have regrets later.